<ui version="4.0" >
 <class>FtpClientWindowClass</class>
 <widget class="QDialog" name="FtpClientWindowClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>466</width>
    <height>549</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>ftpClient</string>
  </property>
  <property name="modal" >
   <bool>true</bool>
  </property>
  <layout class="QHBoxLayout" >
   <property name="spacing" >
    <number>6</number>
   </property>
   <property name="leftMargin" >
    <number>9</number>
   </property>
   <property name="topMargin" >
    <number>9</number>
   </property>
   <property name="rightMargin" >
    <number>9</number>
   </property>
   <property name="bottomMargin" >
    <number>9</number>
   </property>
   <item>
    <layout class="QVBoxLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <property name="leftMargin" >
      <number>0</number>
     </property>
     <property name="topMargin" >
      <number>0</number>
     </property>
     <property name="rightMargin" >
      <number>0</number>
     </property>
     <property name="bottomMargin" >
      <number>0</number>
     </property>
     <item>
      <widget class="QListWidget" name="listWidget" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QGridLayout" >
       <property name="leftMargin" >
        <number>0</number>
       </property>
       <property name="topMargin" >
        <number>0</number>
       </property>
       <property name="rightMargin" >
        <number>0</number>
       </property>
       <property name="bottomMargin" >
        <number>0</number>
       </property>
       <property name="horizontalSpacing" >
        <number>6</number>
       </property>
       <property name="verticalSpacing" >
        <number>6</number>
       </property>
       <item row="2" column="2" >
        <widget class="QLabel" name="passWordLab" >
         <property name="text" >
          <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;/head>&lt;body style=" white-space: pre-wrap; font-family:宋体; font-size:9pt; font-weight:400; font-style:normal; text-decoration:none;">&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-size:10pt; color:#0000ff;">密码：&lt;/span>&lt;/p>&lt;/body>&lt;/html></string>
         </property>
         <property name="buddy" >
          <cstring>passWordlineEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="2" >
        <widget class="QLabel" name="ftpServerPortLab" >
         <property name="text" >
          <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;/head>&lt;body style=" white-space: pre-wrap; font-family:宋体; font-size:9pt; font-weight:400; font-style:normal; text-decoration:none;">&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-size:10pt; color:#0000ff;">端口：&lt;/span>&lt;/p>&lt;/body>&lt;/html></string>
         </property>
         <property name="buddy" >
          <cstring>ftpServerPortLineEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="0" >
        <widget class="QLabel" name="ftpServerLab" >
         <property name="text" >
          <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;/head>&lt;body style=" white-space: pre-wrap; font-family:宋体; font-size:9pt; font-weight:400; font-style:normal; text-decoration:none;">&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-size:10pt; color:#0000ff;">FTP服务器：&lt;/span>&lt;/p>&lt;/body>&lt;/html></string>
         </property>
         <property name="buddy" >
          <cstring>ftpServerLineEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="0" >
        <widget class="QLabel" name="userNameLab" >
         <property name="text" >
          <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;/head>&lt;body style=" white-space: pre-wrap; font-family:宋体; font-size:9pt; font-weight:400; font-style:normal; text-decoration:none;">&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-size:10pt; color:#0000ff;">用户名：&lt;/span>&lt;/p>&lt;/body>&lt;/html></string>
         </property>
         <property name="buddy" >
          <cstring>userNamelineEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="4" >
        <widget class="QLabel" name="statusLab" >
         <property name="text" >
          <string/>
         </property>
        </widget>
       </item>
       <item row="2" column="1" >
        <widget class="QLineEdit" name="userNamelineEdit" >
         <property name="text" >
          <string>anonymous</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" >
        <widget class="QLineEdit" name="ftpServerLineEdit" >
         <property name="text" >
          <string>ftp.ntu.edu.tw</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3" >
        <widget class="QLineEdit" name="ftpServerPortLineEdit" >
         <property name="text" >
          <string>21</string>
         </property>
        </widget>
       </item>
       <item row="2" column="3" >
        <widget class="QLineEdit" name="passWordlineEdit" >
         <property name="text" >
          <string>anonymous</string>
         </property>
         <property name="echoMode" >
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" >
       <property name="spacing" >
        <number>6</number>
       </property>
       <property name="leftMargin" >
        <number>0</number>
       </property>
       <property name="topMargin" >
        <number>0</number>
       </property>
       <property name="rightMargin" >
        <number>0</number>
       </property>
       <property name="bottomMargin" >
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="connectPB" >
         <property name="text" >
          <string>连接</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cdUpDirPB" >
         <property name="enabled" >
          <bool>false</bool>
         </property>
         <property name="text" >
          <string>  上级目录</string>
         </property>
         <property name="icon" >
          <iconset resource="ftp.qrc" >:/images/cdtoparent.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="downLoadPB" >
         <property name="enabled" >
          <bool>false</bool>
         </property>
         <property name="text" >
          <string>下载</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="quitPB" >
         <property name="text" >
          <string>退出</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <tabstops>
  <tabstop>ftpServerLineEdit</tabstop>
  <tabstop>ftpServerPortLineEdit</tabstop>
  <tabstop>userNamelineEdit</tabstop>
  <tabstop>passWordlineEdit</tabstop>
  <tabstop>connectPB</tabstop>
  <tabstop>cdUpDirPB</tabstop>
  <tabstop>downLoadPB</tabstop>
  <tabstop>quitPB</tabstop>
  <tabstop>listWidget</tabstop>
 </tabstops>
 <resources>
  <include location="ftp.qrc" />
 </resources>
 <connections/>
</ui>
